### ISSUE TITLE
Option to disable layout resizable

### ISSUE THREAD

**Issue by nhuttrung** [angular-ui/ui-layout#138]
Could you provide an option to disable layout resizable?
For example, in demo at http://angular-ui.github.io/ui-layout/, it would be nicer to make the north and south pane non-resizable.

Thanks,

**Comment by SomeKittens** [comment]
I'm not sure what you want - the purpose of this module is to allow for things to be resized.  Why not just use a div?

**Comment by ice-blaze** [comment]
In a more general case, disable all resizable bar. The purpose would be to define a nice layout, and then set it in a static state. I hope it gives more sense to the the request.

**Comment by SomeKittens** [comment]
So you want to be able to toggle the resizing on and off?  That makes more sense.

**Comment by ice-blaze** [comment]
Yes exactly. But I think that nhuttrung wanted something where you can select precisily which bar to freez. I guess freez the whole layout is simple, but only freez some bars is a bit more tricky to implement. In my case the complete freez would be enough.

**Comment by nhuttrung** [comment]
Thanks @SomeKittens for changing the title. The new title is exactly what I mean.
